css/styles.css

```diff
@@ -480,27 +481,32 @@ textarea {
 }
 
 @media (min-width: 780px) {
-  header {
+  .nav-menu {
     position: fixed;
+    z-index: 100;
+    height: 104px;
     display: flex;
-    width: 100%;
+    width: 1393px;
```

Comment:
kindly see below the current display of your project, your current changes might have disrupted your page outlook

⬇️   this is how your page shows at 760px view you can avoid this by using flexible width e.g width: 100% in large containers instead of a fixed width with a px.

![image](https://user-images.githubusercontent.com/43564324/165983613-65644598-d7fa-4e5d-881f-c7d751afb3ac.png)

you can also see that the link to your icons image is broken kindly add a dot at the beginning of the link, e.g `src="./css/VectorM.png"`